qiskit_ibm_provider/transpiler/passes/scheduling/utils.py

```diff
@@ -256,3 +265,20 @@ def _patch_key(self, key: InstrKey, duration: int, unit: str) -> None:
             self.duration_by_name_qubits[(key[0], key[1])] = (duration, unit)
 
         self.duration_by_name_qubits_params[key] = (duration, unit)
+
+    def _get_odd_cycle_correction(self) -> int:
+        """Determine the amount of the odd cycle correction to apply
+        For devices with short gates with odd lenghts we add an extra 16dt to the measurement
+
+        TODO: Eliminate the need for this correction
+        """
+        key_pulse = "sx"
+        key_qubit = 0
+        try:
+            key_duration = self.get(key_pulse, key_qubit, "dt")
+        except TranspilerError:
+            key_duration = 160  # keyPulse gate not found
```

Comment:
Can we log a warning here in this scenario?

Reply:
Added in https://github.com/Qiskit/qiskit-ibm-provider/pull/576/commits/4bc34f18efdde8c918eb472abe69a54a414fa4d2